src/pad.js

```diff
@@ -27,9 +27,9 @@ const utils = require('./utils')
  * @param {NamedNode} author - The author of text being displayed
  * @returns {String} The CSS color generated, constrained to be light for a background color
  */
-UI.pad.lightColorHash = function (author) {
-  var hash = function (x) {
-    return x.split('').reduce(function (a, b) {
+UI.pad.lightColorHash = function(author) {
+  var hash = function(x) {
+    return x.split('').reduce(function(a, b) {
```

Comment:
Do we have a lint rule enabled for this spacing? If not, we should!

Reply:
Ah wait, we do. Try running `npm run lint` on your code, you'll see it complain:
<img width="745" alt="Screenshot 2020-02-10 09 01 29" src="https://user-images.githubusercontent.com/408412/74131139-fa774000-4be3-11ea-8666-ede93e418f83.png">